api.py

```diff
@@ -189,7 +189,7 @@ def post(self):
         # validate bands and convert the correct format
         if response['status'] == 'ok':
             if 'band' in arguments:
-                bands = arguments['band'].replace('[','').replace(']','')
+                bands = arguments['band'].replace('[','').replace(']','').replace("'",'').replace(' ','')
```

Comment:
yes, it will change to API. It may not work with the old API. I just thought this way seems more intuitive.